### PR TITLE
KTL-949 Fix prototype pollution in webpack loader-utils

### PR DESCRIPTION
**Pull request by nikpachoo**
https://youtrack.jetbrains.com/issue/KTL-949/Fix-prototype-pollution-in-webpack-loader-utils

This change aims to fix [this issue](https://github.com/JetBrains/kotlin-web-site/security/dependabot/24) by removing outdated packages.

### CODE REVIEW

**Comment by nikpachoo** [comment]
Please don't merge it. The version dropdown is broken - https://branch-ktl-949-fix-npm-dep.kotlin-web-site.labs.jb.gg/api/latest/jvm/stdlib/

File: static/js/com/dropdown/index.js [hunk 1]
```diff
@@ -112,4 +110,19 @@ Dropdown.prototype.select = function (index, emit) {
   emit && this._emitter.emit('select', selectedValue);
 };
 
+function renderTemplate(dropdown) {
+  let items = '';
+
+  for (const [key, value] of Object.entries(dropdown.items)) {
+    items += `<div class="dropdown-item js-item" data-value="${key}">${value}</div>`;
```
Comment (zoobestik) [inline static/js/com/dropdown/index.js:117]: @nikpachoo, I'm not sure how it works in twig, but it looks like unsafe inline attr and html values without escaping. Do you know how we use it? Version selector?

**Review by zoobestik — CHANGES_REQUESTED** [review]

File: static/js/com/dropdown/index.js (hunk 1 above)
Comment (nikpachoo) [reply static/js/com/dropdown/index.js:117]: Yeah, the version selector uses the dropdown component. The values are [here](https://github.com/JetBrains/kotlin-web-site/blob/cf8b857afb94c15306d75c97f764d77e2bcf538d/static/js/page/api/api.js#L187).

Comment (nikpachoo) [reply static/js/com/dropdown/index.js:117]: @zoobestik, please take a look

Comment (zoobestik) [reply static/js/com/dropdown/index.js:117]: For me it's ok if it's only in for api version selector. 👍

**Review by zoobestik — APPROVED** [review]

**Pull request merged**